<h2>Description</h2>
<p>
Human caudate cell type open chromatin regions and ortholog predictions in 240 mammals.
</p>
<p>
  This track shows the ATAC-seq open chromatin regions from human
  postmortem brain scATAC-seq. These regions were obtained from Corces et
  al. Nature Genetics GSE147672. MSN_D2 caudate cell type.
</p>

<h2>Methods</h2>
<p>
  We used ArchR to cluster caudate scATAC-seq samples to resolve medium
  spiny neuron subtypes (D1, D2, striatonigral), interneurons, and four
  glia cell types from the human caudate. We then mapped reproducible
  peaks of each cell type across 240 mammals using the
  <a href="https://zoonomiaproject.org/">Zoonomia Consortium</a>
  multi-species cactus alignment of 241 mammals (including
  humans, hg38). We used cross-species cell type-specific machine
  learning models to predict open chromatin state of the DNA orthologs of
  each mappable peak to these species.
</p>

<h2>Credits</h2>
<p>
  The Pfenning Lab, as well as the UCSC Genome Browser, provided the
  input data.
</p>

<h2>Contact</h2>

<p>
  Please contact <a href="mailto:apfenning@cmu.edu">Andreas Pfenning</a> or
  <a href="mailto:badoi.phan@pitt.edu">BaDoi Phan</a> with questions.
</p>

<h2>References</h2>

<p>
Granja JM, Corces MR, Pierce SE, Bagdatli ST, Choudhry H, Chang HY, Greenleaf WJ.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33633365" target="_blank">
ArchR is a scalable software package for integrative single-cell chromatin accessibility
analysis</a>.
<em>Nat Genet</em>. 2021 Mar;53(3):403-411.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33633365" target="_blank">33633365</a>;
PMC: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8012210/" target="_blank">PMC8012210</a>;
DOI: <a href="https://doi.org/10.1038/s41588-021-00790-6" target="_blank">10.1038/s41588-021-00790-6</a>
</p>

<p>
Corces MR, Shcherbina A, Kundu S, Gloudemans MJ, Fr&#233;sard L, Granja JM, Louie BH, Eulalio T,
Shams S, Bagdatli ST <em>et al</em>.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33106633" target="_blank">
Single-cell epigenomic analyses implicate candidate causal variants at inherited risk loci for
Alzheimer's and Parkinson's diseases</a>.
<em>Nat Genet</em>. 2020 Nov;52(11):1158-1168.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33106633" target="_blank">33106633</a>;
PMC: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7606627/" target="_blank">PMC7606627</a>;
DOI: <a href="https://doi.org/10.1038/s41588-020-00721-x" target="_blank">10.1038/s41588-020-00721-x</a>
</p>

<p>
Zoonomia Consortium.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33177664" target="_blank">
A comparative genomics multitool for scientific discovery and conservation</a>.
<em>Nature</em>. 2020 Nov;587(7833):240-245.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33177664" target="_blank">33177664</a>;
PMC: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7759459/" target="_blank">PMC7759459</a>;
DOI: <a href="https://doi.org/10.1038/s41586-020-2876-6" target="_blank">10.1038/s41586-020-2876-6</a>
</p>

<p>
  <!-- BEGIN_IN_PREP -->
   Phan, BN, Lawler AJ, He J,  et al. Fine-mapping candidate neurological
   disorder-associated regulatory variants using cell type-aware
   comparative genomics, in preparation.
  <!-- END_IN_PREP -->
</p>


