<h1> TACIT Open Chromatin and Open Chromatin Prediction Tracks (2022-5-10)
</h1>

<h2>Tracks and Descriptions</h2>

<dl>
  <ul>
    <li>
    <em>MusMusculusMotorCortex</em> - Motor cortex open chromatin for house mouse measured from ATAC-seq (data from GSE161374).</li>
    <li>
    <em>MusMusculusOrthologMotorCortexPred</em> - Predicted motor cortex open chromatin for house mouse motor cortex open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/MotorCortex/MultiSpecies/">TACIT supplement website multi-species motor cortex model</a>).</li>
    <li>
    <em>MacacaMulattaMotorCortex</em> - Motor cortex open chromatin for rhesus macaque measured from ATAC-seq (data from GSE159815).</li>
    <li>
    <em>MacacaMulattaOrthologMotorCortexPred</em> - Predicted motor cortex open chromatin for rhesus macaque motor cortex open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/MotorCortex/MultiSpecies/">TACIT supplement website multi-species motor cortex model</a>).</li>
	 <li>
    <em>RattusNorvegicusMotorCortex</em> - Motor cortex open chromatin for Norway rat measured from ATAC-seq (data from GSE159815).</li>
    <li>
    <em>RattusNorvegicusOrthologMotorCortexPred</em> - Predicted motor cortex open chromatin for Norway rat motor cortex open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/MotorCortex/MultiSpecies/">TACIT supplement website multi-species motor cortex model</a>).</li>
    <li>
    <em>RousettusAegyptiacusMotorCortex</em> - Motor cortex open chromatin for Egyptian fruit bat measured from ATAC-seq (data from GSE187366).</li>
    <li>
    <em>RousettusAegyptiacusOrthologMotorCortexPred</em> - Predicted motor cortex open chromatin for Egyptian fruit bat motor cortex open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/MotorCortex/MultiSpecies/">TACIT supplement website multi-species motor cortex model</a>).</li>
    <li>
    <em>MusMusculusLiver</em> - Liver open chromatin for house mouse measured from ATAC-seq (data from GSE159815).</li>
    <li>
    <em>MusMusculusOrthologLiverPred</em> - Predicted liver open chromatin for house mouse liver open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/Liver/MultiSpecies/">TACIT supplement website multi-species liver model</a>).</li>
	<li>
    <em>MacacaMulattaLiver</em> - Liver open chromatin for rhesus macaque measured from ATAC-seq (data from GSE159815).</li>
    <li>
    <em>MacacaMulattaOrthologLiverPred</em> - Predicted liver open chromatin for rhesus macaque liver open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/Liver/MultiSpecies/">TACIT supplement website multi-species liver model</a>).</li>
	<li>
    <em>RattusNorvegicusLiver</em> - Liver open chromatin for Norway rat measured from ATAC-seq (data from GSE159815).</li>
    <li>
    <em>RattusNorvegicusOrthologLiverPred</em> - Predicted liver open chromatin for Norway rat liver open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/Liver/MultiSpecies/">TACIT supplement website multi-species liver model</a>).</li>
	<li>
    <em>BosTaurusLiver</em> - Liver open chromatin for cow measured from ATAC-seq (data from GSE158414).</li>
    <li>
    <em>BosTaurusOrthologLiverPred</em> - Predicted liver open chromatin for cow liver open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/Liver/MultiSpecies/">TACIT supplement website multi-species liver model</a>).</li>
	<li>
    <em>SusScrofaLiver</em> - Liver open chromatin for pig measured from ATAC-seq (data from GSE158414).</li>
    <li>
    <em>SusScrofaOrthologLiverPred</em> - Predicted liver open chromatin for pig liver open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/Liver/MultiSpecies/">TACIT supplement website multi-species liver model</a>).</li>
	<li>
    <em>MusMusculusPV</em> - Motor cortex PV+ neuron open chromatin for house mouse measured from snATAC-seq (data from NeMO dat-wywv153).</li>
    <li>
    <em>MusMusculusOrthologPVPred</em> - Predicted PV open chromatin for house mouse PV open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/PVInterneurons/MultiSpecies/">TACIT supplement website multi-species PV+ interneuron model</a>).</li>
	<li>
    <em>HomoSapiensPV</em> - Motor cortex PV+ neuron open chromatin for human measured from SNARE-seq2 (data from NeMO dat-ek5dbmu).</li>
    <li>
    <em>HomoSapiensOrthologPVPred</em> - Predicted PV open chromatin for human PV open chromatin region orthologs (machine learning model from <a target="_blank" href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/CNNs/PVInterneurons/MultiSpecies/">TACIT supplement website multi-species PV+ interneuron model</a>).</li>
  </ul>
</dl>

<h2>Methods</h2>
<p>
For previously published bulk ATAC-seq datasets, we used the irreproducible discovery rate (IDR) optimal peaks.  For the mouse snATAC-seq data, we mapped the reads with bowtie2.  For the human SNARE-seq2 data, we filtered the reads with filtering the snATAC-seq reads with alignmentSieve.  For both snATAC-seq datasets, we obtained reproducible peaks with ArchR and used the peaks from the cluster of cells that had been labeled as PV+ interneurons.  We obtained enhancers using bedtools and Gencode transcription start sites mapped to the species with the data combined with, for non-mouse and non-human species, RefSeq annotations in that species.  We obtained predictions at orthologs by mapping the motor cortex enhancers to 221 Boreoeutherian mammals in the Zoonomia Cactus Alignment using halLiftover and HALPER and then making predictions for the peak summit orthologs +/- 250bp using the machine learning model for the tissue or cell type.</p>

<h2>Data Access</h2>
<p>
The  data for these tracks can be accessed at the <a href="http://daphne.compbio.cs.cmu.edu/files/ikaplow/TACITSupplement/PredictionBrowserTracks/" target="_blank">TACIT supplement website prediction tracks</a>.</p>

<h2>Credits</h2>
<p>
The Pfenning Lab and the UCSC Genome Browser provided the input data. </p>

<h2>References</h2>
<div id="refs" class="references" role="doc-bibliography">
<div id="ref-mouse-cortex-data">
<p>Srinivasan, Chaitanya, BaDoi N. Phan, Alyssa J. Lawler, Easwaran Ramamurthy, Michael Kleyman, Ashley R. Brown, et al. 2021. "Addiction-Associated Genetic Variants Implicate Brain Cell Type- and Region-Specific Cis-Regulatory Elements in Addiction Neurobiology." <em>Journal of Neuroscience.</em> 41 (43): 9008-9030. <a href="https://www.jneurosci.org/content/41/43/9008.long">https://www.jneurosci.org/content/41/43/9008.long</a>.</p>
</div>
<div id="ref-data">
<p>Wirthlin, Morgan, Irene M. Kaplow, Alyssa J. Lawler, Jing He, BaDoi N. Phan, et al. 2020. "The Regulatory Evolution of the Primate Fine-Motor System." <em>bioRxiv.</em> <a href="https://www.biorxiv.org/content/10.1101/2020.10.27.356733v1">https://www.biorxiv.org/content/10.1101/2020.10.27.356733v1</a>.</p>
</div>
<div id="ref-bat-data">
<p>Wirthlin, Morgan, Xiaomeng Zhang, Irene M. Kaplow, Daniel E. Schaffer, Alyssa J. Lawler, Tobias A. Schmid, et al. "Vocal learning-associated convergent evolution in mammalian regulatory elements and proteins."</p>
</div>
<div id="ref-mouse-single-cell-data">
<p>Li, Yang E., Sebastian Preissl, Xiaomeng Hou, Ziyang Zhang, Kai Zhang, Yunjiang Qiu, et al. 2021. "An atlas of gene regulatory elements in adult mouse cerebrum." <em>Nature.</em> 598 (7879): 129-136. <a href="https://www.nature.com/articles/s41586-021-03604-1">https://www.nature.com/articles/s41586-021-03604-1</a>.</p>
</div>
<div id="ref-human-single-cell-data">
<p>Bakken, Trygve E., Nikolas L. Jorstad, Qiwen Hu, Blue B. Lake, Wei Tian, Brian E. Kalmbach, et al. 2021. "Comparative cellular analysis of motor cortex in human, marmoset and mouse." <em>Nature.</em> 598 (7879): 111-119. <a href="https://www.nature.com/articles/s41586-021-03465-8">https://www.nature.com/articles/s41586-021-03465-8</a>.</p>
</div>
<div id="ref-read-mapping">
<p>Langmead, Ben and Steven L Salzberg. 2012. “Fast gapped-read alignment with Bowtie 2.” <em>Nature Methods.</em> 9 (4): 357-359. <a href="https://www.nature.com/articles/nmeth.1923">https://www.nature.com/articles/nmeth.1923</a>.</p>
</div>
<div id="ref-data-processing">
<p>“ENCODE ATAC-seq pipeline.” <a href="https://github.com/ENCODE-DCC/atac-seq-pipeline">https://github.com/ENCODE-DCC/atac-seq-pipeline</a>.</p>
</div>
<div id="ref-single-cell-data-processing">
<p>Granja, Jeffrey M., M. Ryan Corces, Sarah E. Pierce, S. Tansu Bagdatli, Hani Choudhry, Howard Y. Chang, et al. 2021. “ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis.” <a href="https://www.nature.com/articles/s41588-021-00790-6">https://www.nature.com/articles/s41588-021-00790-6</a>.</p>
</div>
<div id="ref-read-filtering">
<p>Ramírez, Fidel, Devon P. Ryan, Björn Grüning, Vivek Bhardwaj, Fabian Kilpert, Andreas S. Richter, et al. 2016. “deepTools2: a next generation web server for deep-sequencing data analysis.” <em>Nucleic Acids Research.</em> 44 (W1): W160-W165. <a href="https://academic.oup.com/nar/article/44/W1/W160/2499308?login=true">https://academic.oup.com/nar/article/44/W1/W160/2499308?login=true</a>.</p>
</div>
<div id="ref-data-post-processing">
<p>Kaplow, Irene M., Daniel E. Schaffer, Morgan E. Wirthlin, Alyssa J. Lawler, Ashley R. Brown, Michael Kleyman, et al. 2022. "Inferring mammalian tissue-specific regulatory conservation by predicting tissue-specific differences in open chromatin." <em>BMC Genomics.</em> 23 (1): 291. </p>
</div>
<div id="ref-bedtools">
<p>Quinlan, Aaron R. and Ira M. Hall. 2010. “BEDTools: a flexible suite of utilities for comparing genomic features.” <em>Bioinformatics</em> 26 (6): 841-842. <a href="https://academic.oup.com/bioinformatics/article/26/6/841/244688?login=true">https://academic.oup.com/bioinformatics/article/26/6/841/244688?login=true</a>.</p>
</div>
<div id="ref-gencode">
<p>Frankish, Adam, Mark Diekhans, Anne-Maud Ferreira, Rory Johnson, Irwin Jungreis, Jane Loveland, et al. 2019. “GENCODE reference annotation for the human and mouse genomes.” <em>Nucleic Acids Research</em> 47 (D1): D766-D773. <a href="https://academic.oup.com/nar/article/47/D1/D766/5144133?login=true">https://academic.oup.com/nar/article/47/D1/D766/5144133?login=true</a>.</p>
</div>
<div id="ref-refseq">
<p>O'Leary, Nuala A., Matthew W. Wright, J. Rodney Brister, Stacy Ciufo, Diana Haddad, Rich McVeigh, et al. 2016. “Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation.” <em>Nucleic Acids Research</em> 44 (D1): D733-D745. <a href="https://academic.oup.com/nar/article/44/D1/D733/2502674?login=true">https://academic.oup.com/nar/article/44/D1/D733/2502674?login=true</a>.</p>
</div>
<div id="ref-tacit">
<p>Kaplow, Irene M., Alyssa J. Lawler, Daniel E. Schaffer, Chaitanya Srinivasan, Morgan E. Wirthlin, BaDoi N. Phan, et al. In review. “Relating enhancer genetic variation across mammals to complex phenotypes using machine learning.”</p>
</div>
<div id="ref-cactus">
<p>Armstrong, Joel, Glenn Hickey, Mark Diekhans, Ian T. Fiddes, Adam M Novak, Alden Deran, et al. 2020. “Progressive Cactus is a multiple-genome aligner for the thousand-genome era.” <em>Nature</em> 587 (7833): 246-251. <a href="https://www.nature.com/articles/s41586-020-2871-y">https://www.nature.com/articles/s41586-020-2871-y</a>.</p>
</div>
<div id="ref-haliftover">
<p>Hickey, Glenn, Benedict Paten, Dent Earl, Daniel Zerbino, David Haussler. 2013. “HAL: a hierarchical format for storing and analyzing multiple genome alignments.” <em>Bioinformatics</em> 29 (10): 1341-1342. <a href="https://academic.oup.com/bioinformatics/article/29/10/1341/256598?login=true">https://academic.oup.com/bioinformatics/article/29/10/1341/256598?login=true</a>.</p>
</div>
<div id="ref-halper">
<p>Zhang, Xiaoyu, Irene M. Kaplow, Morgan Wirthlin, Tae Yoon Park, Andreas R. Pfenning. 2020. “HALPER facilitates the identification of regulatory element orthologs across species.” <em>Bioinformatics</em> 36 (15): 4339-4340. <a href="https://academic.oup.com/bioinformatics/article/36/15/4339/5837107?login=true">https://academic.oup.com/bioinformatics/article/36/15/4339/5837107?login=true</a>.</p>
</div>
<div id="ref-zoonomia">
<p>Christmas, Matthew J., Irene M. Kaplow, Diane P. Genereux, Michael X. Dong, Graham M. Hughes, Xue Li, et al. In review. “Evolutionary constraint and innovation across hundreds of placental mammals.”</p>
</div>
</div>

